docs/docusaurus/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_azure_blob_storage.md

```diff
@@ -88,25 +83,25 @@ Finished[#############################################################]  100.000
 "lastModified": "2021-03-06T10:55:33+00:00"
 }
 ```
+To learn more about other methods that are available to copy Expectation JSON files into Azure Blob Storage, see [Introduction to Azure Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction).
 
+## 5. Confirm that the new Expectation Suites have been added
 
-### 5. Confirm that the new Expectation Suites have been added
-
-If you followed the optional step to copy your existing Expectation Suites to Azure blob storage, you can confirm that Great Expectations can find them by running the following Python code:
+If you copied your existing Expectation Suites to Azure Blob Storage, run the following Python command to confirm that Great Expectations can find them:
 
 ```python
```

Comment:
Another code snippet that should be getting pulled from an example script under test.

Reply:
@Rachel-Reverie this has been noted.